packages/slate-base64-serializer/package.json

```diff
@@ -4,34 +4,37 @@
   "version": "0.2.21",
   "license": "MIT",
   "repository": "git://github.com/ianstormtaylor/slate.git",
-  "main": "./lib/index.js",
+  "main": "./dist/cjs/slate-base64-serializer.js",
+  "module": "./dist/es/slate-base64-serializer.js",
+  "browser": "./dist/umd/slate-base64-serializer.js",
+  "browserMin": "./dist/umd/slate-base64-serializer.min.js",
   "files": [
-    "dist/",
-    "lib/"
+    "dist/cjs/slate-base64-serializer.js",
```

Comment:
Why not just `"dist/"` here instead of having to list them all out?

Reply:
Currently the way I have it configured results in sourcemaps also being put in the `dist/` folder when in development mode, so I listed it out to avoid accidentally shipping sourcemaps along with the package